src/metakb/cli.py

```diff
@@ -7,401 +7,366 @@
-    @click.option(
-        "--update_cached",
-        "-u",
-        is_flag=True,
-        default=False,
-        required=False,
-        help=(
-            "`True` if civicpy cache should be updated. Note this will take serveral"
-            "minutes. `False` if local cache should be used"
-        ),
-    )
-    async def update_metakb_db(
-        db_url: str,
-        db_username: str,
-        db_password: str,
-        load_normalizers_db: bool,
-        force_load_normalizers_db: bool,
-        normalizers_db_url: str,
-        load_latest_cdms: bool,
-        load_target_cdm: Optional[Path],
-        load_latest_s3_cdms: bool,
-        update_cached: bool,
-    ) -> None:
-        """Execute data harvest and transformation from resources and upload
-        to graph datastore.
-        """
-        if sum([load_latest_cdms, bool(load_target_cdm), load_latest_s3_cdms]) > 1:
-            CLI()._help_msg(
-                "Error: Can only use one of `--load_latest_cdms`, "
-                "`--load_target_cdm`, `--load_latest_s3_cdms`."
+    if not any([load_latest_cdms, load_target_cdm, load_latest_s3_cdms]):
+        if force_load_normalizers_db:
+            if normalizers_db_url:
+                for env_var_name in [
+                    "GENE_NORM_DB_URL",
+                    "THERAPY_NORM_DB_URL",
+                    "DISEASE_NORM_DB_URL",
+                ]:
+                    environ[env_var_name] = normalizers_db_url
+
+            _load_normalizers_db()
+
+        _harvest_sources(update_cached)
+        await _transform_sources()
+
+    # Load neo4j database
+    start = timer()
+    echo_info("Loading neo4j database...")
+
+    g = Graph(uri=db_url, credentials=(db_username, db_password))
+
+    if load_target_cdm:
+        g.load_from_json(load_target_cdm)
+    else:
+        version = _retrieve_s3_cdms() if load_latest_s3_cdms else None
+        g.clear()
+
+        for src in sorted({v.value for v in SourceName.__members__.values()}):
+            pattern = (
+                f"{src}_cdm_{version}.json"
+                if version is not None
+                else f"{src}_cdm_*.json"
             )
+            globbed = (APP_ROOT / "data" / src / "transform").glob(pattern)
+
+            try:
+                path = sorted(globbed)[-1]
+            except IndexError as e:
+                msg = f"No valid transform file found matching pattern: {pattern}"
+                raise FileNotFoundError(msg) from e
+
+            g.load_from_json(path)
+
+    g.close()
+    end = timer()
+    echo_info(f"Successfully loaded neo4j database in {(end - start):.5f} s\n")
+
+
+def _retrieve_s3_cdms() -> str:
+    """Retrieve most recent CDM files from VICC S3 bucket.
+    Expects to find files in a path like the following:
+        s3://vicc-metakb/cdm/20220201/civic_cdm_20220201.json.zip
+
+    :raise: ResourceLoadException if S3 initialization fails
+    :raise: FileNotFoundError if unable to find files matching expected
```

Comment:
looks like we had goofed this when we originally wrote it 

```suggestion
    :raise ResourceLoadException: if S3 initialization fails
    :raise FileNotFoundError:  if unable to find files matching expected
```